src/item/Item.php

```diff
@@ -222,6 +226,14 @@ public function setCanDestroy(array $canDestroy) : void{
 		}
 	}
 
+	public function keepOnDeath() : bool{
```

Comment:
Please add documentation for what this signifies. I know it seems obvious in the context of the PR, but someone who randomly stumbles across this won't know that this is specifically to make players retain the item (it won't affect other entities with inventories, for example).